static/js/user_status_ui.js

```diff
@@ -84,6 +86,17 @@ export function clear_message() {
 }
 
 export function initialize() {
+    const rendered_set_status_overlay = render_set_status_overlay();
+    $(".app").append(rendered_set_status_overlay);
+
+    $(".user-status-value").on("click", (e) => {
```

Comment:
Can you add a follow-up commit making this a body-on-`user-status-value` style change?  That'd be much safer, as the current implementation relies on the HTML element having been rendered before we call this initialize function.

Reply:
The `#clear_status_message_button` handler, below, would also benefit from this treatment.